### ISSUE TITLE
query projections

### ISSUE THREAD

**Issue by nevf** [YuriSolovyov/Dexie-mongoify#7]
@YuriSolovyov Are there any plans to support query projections so we can just get specific document fields returned. See: https://docs.mongodb.org/manual/reference/method/db.collection.find/

**Comment by YurySolovyov** [comment]
@nevf in case of mongoify, I'm not sure this would give you better performance, if you need this for speed reason.

Thanks for opening the issue, it is now on the TODO list.

**Comment by nevf** [comment]
@YuriSolovyov No, nothing to do with performance. Just another step in Mongo compatibility. Projections can also simplify downstream code, by just returning specific properties.
